enclave-tls/samples/enclave-tls-client/client.c

```diff
@@ -38,20 +39,49 @@ static sgx_enclave_id_t load_enclave(bool debug_enclave)
 	int updated = 0;
 	int ret = sgx_create_enclave(ENCLAVE_FILENAME, debug_enclave, &t, &updated, &eid, NULL);
 	if (ret != SGX_SUCCESS) {
-		ETLS_ERR("Failed to load enclave %d\n", ret);
-		return -1;
+		printf("Failed to load enclave %d\n", ret);
+		return 0;
 	}
 
-	ETLS_DEBUG("Success to load enclave id %ld\n", eid);
+	printf("Success to load enclave id %ld\n", eid);
```

Comment:
Why dropping ETLS_DEBUG?

Reply:
In client, not running in SGX enclave, it should use printf. Because ETLS_* is defined in trusted zone.

Reply:
I think this problem is same with below issue.  We can talk this problem there.